nullaway/src/main/java/com/uber/nullaway/dataflow/AccessPathNullnessPropagation.java

```diff
@@ -676,8 +676,8 @@ private static boolean hasPrimitiveType(Node node) {
   }
 
   private static boolean hasNonNullConstantValue(LocalVariableNode node) {
-    if (node.getElement() instanceof VariableElement) {
-      VariableElement element = (VariableElement) node.getElement();
```

Comment:
After upgrading checkerframework `LocalVariableNode::getElement()` returns `VariableElement` so this was `warning: [BadInstanceof] `node.getElement()` is an instance of
VariableElement which is a subtype of VariableElement, so this is
equivalent to a null check.`